ballista/scheduler/scheduler_config_spec.toml

```diff
@@ -36,6 +36,13 @@ type = "ballista_scheduler::state::backend::StateBackend"
 doc = "The configuration backend for the scheduler, possible values: etcd, memory, sled. Default: sled"
 default = "ballista_scheduler::state::backend::StateBackend::Sled"
 
+[[param]]
+abbr = "c"
```

Comment:
Do we need both `config_backend` and `cluster_backend`? Personally, I prefer `cluster_backend`

Reply:
Right now, no. But once part 2 is done, the goal is to be able to have different backends for cluster state and config (eg job) state. But maybe we expose that in the second PR so it is not so confusing.

Reply:
@yahoNanJing are you okay to merge this as is?